go/CHANGELOG.md

```diff
@@ -6,3 +6,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.1.0
 * Added SignWithPem, VerifyWithPem and ExtractJwsHeader.
+
+## 0.1.1
+* Removed verifier's method default value
```

Comment:
This should go above, and we should just call it `# Unreleased` unless you want to do a tag for this. I don't mind if you want to tag or not.

Reply:
I prefer to tag a new patch version, even if it's a minor thing, for consistency with the other implementations.